### PR TITLE
add missing coercions

### LINKED ISSUE #11: SnmpClientError: unknown variable syntax 70, Counter64
As reported by @dfdalamar in #10  We seem to be missing some type coercion for syntax 70, `Counter64`. 

```
[2018-06-13T17:16:01,457][ERROR][logstash.inputs.snmp     ] error invoking walk operation on OID: 1.3.6.1, ignoring {:exception=>#<LogStash::SnmpClientError: unknown variable syntax 70, Counter64>, :backtrace=>["/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp/client.rb:124:in `coerce'", "/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp/client.rb:96:in `block in walk'", "org/jruby/java/proxies/ArrayJavaProxy.java:450:in `each'", "/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp/client.rb:91:in `block in walk'", "/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp/client.rb:80:in `walk'", "/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp.rb:118:in `block in run'", "org/jruby/RubyArray.java:1734:in `each'", "/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp.rb:116:in `block in run'", "org/jruby/RubyArray.java:1734:in `each'", "/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp.rb:106:in `run'", "/usr/share/logstash/logstash-core/lib/logstash/pipeline.rb:514:in `inputworker'", "/usr/share/logstash/logstash-core/lib/logstash/pipeline.rb:507:in `block in start_input'"]}
[2018-06-13T17:16:34,040][ERROR][logstash.inputs.snmp     ] error invoking walk operation on OID: 1.3.6.1, ignoring {:exception=>#<LogStash::SnmpClientError: unknown variable syntax 70, Counter64>, :backtrace=>["/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp/client.rb:124:in `coerce'", "/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp/client.rb:96:in `block in walk'", "org/jruby/java/proxies/ArrayJavaProxy.java:450:in `each'", "/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp/client.rb:91:in `block in walk'", "/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp/client.rb:80:in `walk'", "/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp.rb:118:in `block in run'", "org/jruby/RubyArray.java:1734:in `each'", "/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp.rb:116:in `block in run'", "org/jruby/RubyArray.java:1734:in `each'", "/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp.rb:106:in `run'", "/usr/share/logstash/logstash-core/lib/logstash/pipeline.rb:514:in `inputworker'", "/usr/share/logstash/logstash-core/lib/logstash/pipeline.rb:507:in `block in start_input'"]}
[2018-06-13T17:17:07,562][ERROR][logstash.inputs.snmp     ] error invoking walk operation on OID: 1.3.6.1, ignoring {:exception=>#<LogStash::SnmpClientError: unknown variable syntax 70, Counter64>, :backtrace=>["/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp/client.rb:124:in `coerce'", "/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp/client.rb:96:in `block in walk'", "org/jruby/java/proxies/ArrayJavaProxy.java:450:in `each'", "/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp/client.rb:91:in `block in walk'", "/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp/client.rb:80:in `walk'", "/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp.rb:118:in `block in run'", "org/jruby/RubyArray.java:1734:in `each'", "/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp.rb:116:in `block in run'", "org/jruby/RubyArray.java:1734:in `each'", "/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp.rb:106:in `run'", "/usr/share/logstash/logstash-core/lib/logstash/pipeline.rb:514:in `inputworker'", "/usr/share/logstash/logstash-core/lib/logstash/pipeline.rb:507:in `block in start_input'"]}
[2018-06-13T17:17:39,377][ERROR][logstash.inputs.snmp     ] error invoking walk operation on OID: 1.3.6.1, ignoring {:exception=>#<LogStash::SnmpClientError: unknown variable syntax 70, Counter64>, :backtrace=>["/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp/client.rb:124:in `coerce'", "/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp/client.rb:96:in `block in walk'", "org/jruby/java/proxies/ArrayJavaProxy.java:450:in `each'", "/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp/client.rb:91:in `block in walk'", "/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp/client.rb:80:in `walk'", "/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp.rb:118:in `block in run'", "org/jruby/RubyArray.java:1734:in `each'", "/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp.rb:116:in `block in run'", "org/jruby/RubyArray.java:1734:in `each'", "/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp.rb:106:in `run'", "/usr/share/logstash/logstash-core/lib/logstash/pipeline.rb:514:in `inputworker'", "/usr/share/logstash/logstash-core/lib/logstash/pipeline.rb:507:in `block in start_input'"]}
[2018-06-13T17:18:10,637][ERROR][logstash.inputs.snmp     ] error invoking walk operation on OID: 1.3.6.1, ignoring {:exception=>#<LogStash::SnmpClientError: unknown variable syntax 70, Counter64>, :backtrace=>["/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp/client.rb:124:in `coerce'", "/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp/client.rb:96:in `block in walk'", "org/jruby/java/proxies/ArrayJavaProxy.java:450:in `each'", "/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp/client.rb:91:in `block in walk'", "/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp/client.rb:80:in `walk'", "/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp.rb:118:in `block in run'", "org/jruby/RubyArray.java:1734:in `each'", "/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp.rb:116:in `block in run'", "org/jruby/RubyArray.java:1734:in `each'", "/usr/share/logstash/vendor/bundle/jruby/2.3.0/gems/logstash-input-snmp-0.1.0.beta3/lib/logstash/inputs/snmp.rb:106:in `run'", "/usr/share/logstash/logstash-core/lib/logstash/pipeline.rb:514:in `inputworker'", "/usr/share/logstash/logstash-core/lib/logstash/pipeline.rb:507:in `block in start_input'"]}

```

### PR DESCRIPTION
**Pull request by colinsurprenant**
Fixes #11 
This adds some missing coercions for `BITSTRING`, `COUNTER64` and `NULL`.
Tested manually locally, no spec coverage for this yet (working on it for larger `Client` class specs.

### CODE REVIEW

File: lib/logstash/inputs/snmp/client.rb [hunk 1]
```diff
@@ -104,20 +104,24 @@ def walk(oid, strip_root = 0)
...
     private
 
+    NULL_STRING = "null".freeze
+
     def coerce(variable)
       variable_syntax = variable.getSyntax
       # puts("variable.getSyntaxString=#{variable.getSyntaxString}")
       case variable_syntax
-      when BER::OCTETSTRING
+      when BER::OCTETSTRING, BER::BITSTRING
```
Comment (jsvd) [inline lib/logstash/inputs/snmp/client.rb]: is there any place we can find the whole list of possible options?

Comment (colinsurprenant) [reply lib/logstash/inputs/snmp/client.rb]: https://www.snmp4j.org/doc/org/snmp4j/asn1/BER.html

Comment (webmat) [inline lib/logstash/inputs/snmp/client.rb]: You have a require for `java_import "org.snmp4j.smi.OctetString"`. Do you need to add one for `BitString` as well?

Actually I see you don't have requires for the counters. Ok, I guess I don't have enough context here :-)

Comment (webmat) [reply lib/logstash/inputs/snmp/client.rb]: Yeah I figured out that it's probably coming from the `asn1.BER` import. Never mind :-)

Comment (colinsurprenant) [reply lib/logstash/inputs/snmp/client.rb]: the `OctetString` class is required for the `build_target` method - not needed for the coercion.

**Pull request merged**